docs/src/APIs/canopy/CanopyEnergy.md

```diff
@@ -13,6 +13,14 @@ ClimaLand.Canopy.BigLeafEnergyParameters
 ClimaLand.Canopy.AbstractCanopyEnergyModel
 ```
 
+## Constructor Methods
+
+```@docs
+ClimaLand.Canopy.BigLeafEnergyModel{FT}(;
```

Comment:
@ph-kev when you update [this](https://github.com/CliMA/ClimaLand.jl/blob/6e43b5c3c691bd21802441d88bb886a0b4ed5045/src/standalone/Vegetation/Canopy.jl#L81) to take the toml_dict, could you also update this? thank you! and remove this https://github.com/CliMA/ClimaLand.jl/blob/6e43b5c3c691bd21802441d88bb886a0b4ed5045/src/standalone/Vegetation/canopy_energy.jl#L63

@AlexisRenchon, we can also add https://github.com/CliMA/ClimaLand.jl/blob/6e43b5c3c691bd21802441d88bb886a0b4ed5045/src/standalone/Vegetation/canopy_energy.jl#L67